client/src/app/components/stardust/Output.scss

```diff
@@ -65,6 +68,45 @@
         }
     }
 
+    .unlocks-card {
```

Comment:
I'm not sure that this is the right file for this css. If this is css meant for the `Unlocks.tsx` component, that one is only used in `TransactionPayload.tsx`, which has it's own `scss` file. I don't see what `Output.tsx` (and it's scss) have to do with Unlock component. `TransactionPayload.scss` seams to be a better place than `Output.scss`, which is used wherever the output card is rendered, which is multiple places.

Reply:
Done.